apps/www/graphql/republik-api.schema.graphql

```diff
@@ -100,6 +100,59 @@ type AccessGrantInfo {
   message: String
 }
 
+type AccessGrantStats {
```

Comment:
These types were removed a while ago. Have you generated the schema from apo-prod? Maybe api-love is outdated?